compiler/rustc_llvm/llvm-wrapper/RustWrapper.cpp

```diff
@@ -1658,16 +1658,6 @@ extern "C" void LLVMRustPositionBuilderAtStart(LLVMBuilderRef B,
   unwrap(B)->SetInsertPoint(unwrap(BB), Point);
 }
 
-extern "C" void LLVMRustSetComdat(LLVMModuleRef M, LLVMValueRef V,
-                                  const char *Name, size_t NameLen) {
-  Triple TargetTriple = Triple(unwrap(M)->getTargetTriple());
-  GlobalObject *GV = unwrap<GlobalObject>(V);
-  if (TargetTriple.supportsCOMDAT()) {
```

Comment:
This supports-COMDAT check turns out to be load-bearing; I'm getting lots of test errors on MacOS because MachO doesn't support COMDATs.

Reply:
Ah, fair enough. We can do it inside `llvm::set_comdat` then.

Reply:
I lied about the placement. :^)